admin/services/data-access/pupil-register.data.service.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+const { TYPES } = require('tedious')
+
+const sqlService = require('./sql.service')
+
+const service = {
+  getPupilRegister: async function (schoolId) {
+    const sql = `
+      SELECT
```

Comment:
This could be moved to a view.  Thoughts?

Reply:
@jon-shipley was about to comment on that, it makes sense to me.

Reply:
yep, its pretty big

Reply:
Done